rotkehlchen/tests/unit/decoders/test_yearn.py

```diff
@@ -115,7 +117,7 @@ def test_deposit_yearn_v3(
             asset=Asset('eip155:1/erc20:0xBF319dDC2Edc1Eb6FDf9910E39b37Be221C8805F'),
             balance=Balance(amount=FVal(receive_amount)),
             location_label=user_address,
-            notes=f'Receive {receive_amount} crvUSD-2 yVault after deposit in a yearn-v3 vault',
+            notes=f'Receive {receive_amount} yvcrvUSD-2 after deposit in a yearn-v3 vault',
```

Comment:
this is wrong. The received token is crvUSD-2 yVault. Did you check the transaction that this is decoding? Why would you think changing this is fine?

![Uploading 2024-10-22_21-24.png…]()

Reply:
For this [token](https://etherscan.io/address/0xbf319ddc2edc1eb6fdf9910e39b37be221c8805f), `crvUSD-2 yVault` is the token name, and `yvcrvUSD-2` is the token symbol. The symbol is generally what is used directly after the amount in event notes like this.